src/styles/abstracts/_mixins.scss

```diff
@@ -38,3 +38,244 @@
     text-overflow: ellipsis;
     white-space: nowrap;
 }
+
+@mixin content-padding($line-height) {
+    padding: calc(#{$line-height} * 0.25) 0;
+
+    // Resets defaults
+    margin-block-end: 0;
+    margin-block-start: 0;
+    margin-inline-end: 0;
+    margin-inline-start: 0;
```

Comment:
not sure it matters, but these properties aren't supported by ie11

Reply:
They will be ignored